dags/gcs_xml_to_gcs_csv.py

```diff
@@ -0,0 +1,100 @@
+from airflow import DAG
+from airflow.operators.python import PythonOperator
+from airflow.providers.google.cloud.operators.gcs import GCSListObjectsOperator
+from airflow.providers.google.cloud.hooks.gcs import GCSHook
+from airflow.utils.dates import days_ago
+from datetime import timedelta, datetime
+import csv
+import requests
+import io
+import os
+import re
+import pandas as pd
+import feedparser
+
+
+def gcs_objects_to_process(**kwargs):
+    ti = kwargs["ti"]
+    objects_list = ti.xcom_pull(task_ids='list_gcs_objects')
+    current_date = kwargs['ds'].replace('-', '')                                #The ds represents the execution date so the dag can be
```

Comment:
```suggestion
    current_date = kwargs['ds_nodash']                              #The ds represents the execution date so the dag can be
```

ref : https://airflow.apache.org/docs/apache-airflow/stable/templates-ref.html